test/anchoredinterval.jl

```diff
@@ -289,39 +289,48 @@ using Intervals: canonicalize
         @test !isequal(he, diff_inc)
         @test hash(he) != hash(diff_inc)
 
-        @test !isless(he, hb)
-        @test !isless(hb, he)
-
-        @test isless(he, he + Hour(1))
-        @test isless(hb, hb + Hour(1))
-
-        @test isless(diff_inc, diff_inc + Hour(2))
-        @test !isless(diff_inc, diff_inc + Hour(1))     # Overlap for an instant
+        # Overlap for an instant, so not disjoint
+        @test he < hb
+        @test !(hb < he)
+        @test !(he > hb)
+        @test hb > he
+        @test !(he ≪ hb)
+        @test !(hb ≪ he)
+        @test !(he ≫ hb)
+        @test !(hb ≫ he)
```

Comment:
Just going to mention that `≪̸` (`\NotLessLess`) and `≫̸` (`\NotGreaterGreater`) exist.

Reply:
Unfortunately there doesn't appear to be a not-much-less-than.

Reply:
Whoops, misread that message because `≪̸` and `≮` are basically indistinguishable on my monitor.